packages/table/src/table.tsx

```diff
@@ -542,6 +542,7 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
             this.bodyElement,
             this.grid,
         );
+        this.locator = locator;
 
         const viewportRect = locator.getViewportRect();
         this.setState({ locator, viewportRect });
```

Comment:
Because stuff downstream of it uses it for interactions -- it's still used in render to render the headers and cells and such so it can be used for click and drag interactions. 

I was trying to do a quick fix here, but I do agree there should be a more comprehensive way of doing this -- if so, we should figure out how to set up the interactions in a way that doesn't need the locator passed downstream.